Wordle.Api/Wordle.Api/Controllers/LeaderboardController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Services;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class LeaderboardController : ControllerBase
+{
+	private readonly LeaderboardService _service;
+	public LeaderboardController(LeaderboardService service)
+	{
+		_service = service;
+	}
+
+	[HttpGet("GetScores")]
+	public async Task<List<PlayerDto>> Get()
+	{
+		return await _service.GetTopScoresAsync();
+	}
+
+	[HttpPost("PostScore")]
+	public async Task<PlayerDto> Post(PlayerDto request)
+	{
+		Player player = await _service.PostScoreAsync(request);
+		return new PlayerDto
+		{
+			Name = player.Name,
+			GameCount = player.GameCount,
+			AverageAttempts = player.AverageAttempts
+		};

```

Comment:
I hadn't even considered putting this logic here in the controller, seems nice and elegant.

Reply:
Thanks! I'm still not sure it's the best place, but oh well. I think it works.